src/editor/tests/test_model_procedure.py

```diff
@@ -2,12 +2,21 @@
 from django.contrib.auth.models import User
 from editor.models import Procedure
 
+
 class ProcedureTest(TestCase):
     def setUp(self):
-        test_user = User.objects.create_user('TestUser', 'test@sanaprotocolbuilder.me', 'testpassword')
+        test_user = User.objects.create_user(
+            'TestUser',
+            'test@sanaprotocolbuilder.me',
+            'testpassword'
+        )
         test_user.save()
 
-        Procedure.objects.create(author='tester', title='test procedure', owner=test_user)
+        Procedure.objects.create(
+            author='tester',
```

Comment:
Align these `=` please. Get the alignment Sublime plugin (http://wbond.net/sublime_packages/alignment)